Makefile

```diff
@@ -136,6 +136,8 @@ test-long-ci:
 	@mkdir -p cover
 	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=600s $(pkgs) -run=$(run) -count=$(count)
 
+test-single:
```

Comment:
You don't need this. 

You just need `run` to be previously defined.

then you can run `make test-long pkgs=mypkg run=mytest`

Reply:
what is the error you get when you try and run a single test that is making you need this additional command?

Reply:
`make test-long` always starts with a clean mongo and ends up with a wiped mongo.
This command allows me to play around with the content of the DB both before and after the tests are done. This is incredibly helpful when trying to chase a bug.

My typical use of this command is `make stop-mongo && make start-mongo && make test-single run=TestWithDBSession_RetryOnWriteConflict`.

I am using this command a lot. It's helpful to me. I would like to keep it. What is the rationale for removing it?

Reply:
If it is simply to avoid start and stopping mongo, to keep the mongo state, why not just use `test-long-ci`. That is already refactored.  The only difference is the coverprofile right?

I see the value in running tests that way, so we should support that for sure.

Reply:
Fine, we can remove it.

I don't see any reason not to have it there, as I find it helpful. But I also don't want to battle over it again and again. I'll just keep my local tools in a stash that I'll apply and drop.